File: beginner/pagina02/1041/1041.php
<?php 

    list($a, $b) = explode(" ", readline());
    $a = (float) $a; $b = (float) $b;

    if($a === 0 && $b === 0){echo "Origem\n";}
    else if ($a == 0){echo "Eixo Y\n";}
    else if ($b == 0){echo "Eixo X\n";}
    else if ($b > 0 && $a > 0){echo "Q1\n";}
    else if ($b > 0 && $a < 0){echo "Q2\n";}
    else if ($b < 0 && $a < 0){echo "Q3\n";}
    else if ($b < 0 && $a > 0){echo "Q4\n";}

?>

File: beginner/__1000__/1041/1041.rs
use std::io;

fn main() {

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let data: Vec<f32> = input.trim().split_whitespace().map(|x| x.parse().unwrap()).collect();
    let (a, b) = (data[0], data[1]);

    if a == 0.0 && b == 0.0       { println!("Origem"); }
    else if  a == 0.0               { println!("Eixo Y");}
    else if  b == 0.0               { println!("Eixo X");}
    else if  b > 0.0 && a > 0.0     { println!("Q1");}
    else if  b > 0.0 && a < 0.0     { println!("Q2");}
    else if  b < 0.0 && a < 0.0     { println!("Q3");}
    else if  b < 0.0 && a > 0.0     { println!("Q4");}
    
}